firedrake/slate/__init__.py

```diff
@@ -0,0 +1,3 @@
+from __future__ import absolute_import
+from firedrake.slate.slate import * # noqa
```

Comment:
Please use qualifiers like `# noqa: F401` for the exact error you are suppressing. Your unqualified `noqa` also suppresses errors that should rather be fixed, e.g., you need 2 spaces before inline comment.